demo/build.gradle

```diff
@@ -17,6 +17,10 @@ android {
         abortOnError true
         warningsAsErrors true
     }
+
+    compileOptions {
```

Comment:
This will be a breaking change for library users, it is now required to use this when using exoplayer